CONTRIBUTING.md

```diff
@@ -16,3 +16,12 @@ Contributors must sign-off each commit by adding a `Signed-off-by: ...`
 line to commit messages to certify that they have the right to submit
 the code they are contributing to the project according to the
 [Developer Certificate of Origin (DCO)](https://developercertificate.org/).
+
+
+## Contributing new messages and or packages
+
+To be accepted into common_interfaces the package needs to have been API reviewed and be in active use in a non trivial amount of the ROS ecosystem.
+It's really supposed to represent messages which are commonly used.
+
+On the way to becoming a member of common_interfaces please release a message only package and make it available to the community.
+Once it has matured, been reviewed, tested and possibly iterated upon by early adopters then it can be promoted to be a member of the common_interfaces metapackage.
```

Comment:
@tfoote nit: 
```suggestion
Once it has matured, been reviewed, tested, and possibly iterated upon by early adopters, then it can be promoted to be a member of the `common_interfaces` metapackage.
```